src/ch/ch_conf.c

```diff
@@ -243,3 +239,59 @@ int chStrToInt(const char *str)
 
     return val;
 }
+
+int
+virCHVersionString(const char *str, unsigned long *version,
+                      bool allowMissing)
+{
```

Comment:
Is there some automation wrt. symbol exports? Otherwise, it would seem that this function declaration might need to be moved to an upper place prior its use.

Thanks

Reply:
Moved code upward. Thnx :)